src/pricecypher/datasets.py

```diff
@@ -196,7 +199,7 @@ def get_transactions(
         # NB: the two separate lists of scope value ids are destructured and put in a set. This removes duplicates.
         filter_scope_value_ids = {
             *self._find_scope_value_filters(columns_with_values),
-            *self._get_additional_scope_value_filter_ids(filters, dataset_id),
+            *self._get_additional_scope_value_filter_ids(filters or [], dataset_id),
```

Comment:
Concenptually it behaves pretty similar indeed. It is nothing more than a regular [Boolean Operator though as defined by the spec](https://docs.python.org/3/reference/expressions.html#boolean-operations) though.

Quoted from the docs:
> The expression `x or y` first evaluates _x_; if _x_ is true, its value is returned; otherwise, _y_ is evaluated and the resulting value is returned.